package.json

```
@@ -8,12 +8,12 @@
 		"build": "webpack --node-env production --progress",
 		"dev": "webpack --node-env development --progress",
 		"watch": "webpack --node-env development --progress --watch",
+		"serve": "npx webpack serve --node-env development --progress --allowed-hosts all",
```

```suggestion
		"serve": webpack serve --node-env development --progress --allowed-hosts all",
```
Seems to also working proper

---

Thanks, missed it